.github/workflows/sycl-linux-precommit.yml

```diff
@@ -127,6 +127,24 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - name: E2E tests on Intel PVC/L0 GPU w/ LLVM SPIR-V Backend
+            runner: '["Linux", "pvc"]'
+            image: ghcr.io/intel/llvm/ubuntu2404_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+            target_devices: level_zero:gpu
+            extra_lit_opts: -j 50 --param spirv-backend=True
+          - name: E2E tests on Intel PVC/OCL GPU w/ LLVM SPIR-V Backend
+            runner: '["Linux", "pvc"]'
+            image: ghcr.io/intel/llvm/ubuntu2404_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+            target_devices: opencl:gpu
+            extra_lit_opts: -j 50 --param spirv-backend=True
+          - name: E2E tests on Intel GEN12/OCL CPU w/ LLVM SPIR-V Backend
+            runner: '["Linux", "gen12"]'
+            image: ghcr.io/intel/llvm/ubuntu2404_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+            target_devices: opencl:cpu
+            extra_lit_opts: -j 50 --param spirv-backend=True
```

Comment:
This makes no sense, these runners have desktop CPUs and you'll oversubscribe them.